src/Bridges/ApplicationDI/RoutingExtension.php

```diff
@@ -55,6 +52,13 @@ public function loadConfiguration()
 	public function beforeCompile()
 	{
 		$builder = $this->getContainerBuilder();
+		$router = $builder->getDefinition($this->prefix('router'));
```

Comment:
$router is not needed, or is it? It can be moved back to loadConfiguration() IMHO.